be/src/runtime/local_tablets_channel.cpp

```diff
@@ -539,6 +508,48 @@ void LocalTabletsChannel::_commit_tablets(const PTabletWriterAddChunkRequest& re
     _abort_replica_tablets(request, "", node_id_to_abort_tablets);
 }
 
+void LocalTabletsChannel::_wait_secondary_replicas_commit(const PTabletWriterAddChunkRequest& request) {
+    auto start_wait_ts = butil::gettimeofday_s();
+    bool timeout = false;
+    for (auto& [tablet_id, delta_writer] : _delta_writers) {
+        // Wait util seconary replica commit/abort by primary
+        if (delta_writer->replica_state() == Secondary) {
+            do {
+                auto state = delta_writer->get_state();
+                if (state == kCommitted || state == kAborted || state == kUninitialized) {
+                    break;
+                }
+                // only sleep in bthread
+                bthread_usleep(10000); // 10ms
+
+                auto total_rpc_time = butil::gettimeofday_s() - _rpc_start_ts;
```

Comment:
`_rpc_start_ts` is will be not modify after open?

Reply:
yes, for whole ingestion